external-crates/move/crates/move-vm-runtime/src/jit/execution/translate.rs

```diff
@@ -493,7 +512,10 @@ fn functions(
     package_context: &mut PackageContext,
     signature_cache: &mut SignatureCache,
     module: &mut input::Module,
-) -> PartialVMResult<BTreeMap<SignatureIndex, VMPointer<Type>>> {
+) -> PartialVMResult<(
+    BTreeMap<ConstantPoolIndex, VMPointer<Constant>>,
```

Comment:
Nit: `ConstantCache` and `SignatureCache`

Reply:
The second is not a `SignatureCache`, but I swapped `ConstantCache` in.